c7n/resources/cfn.py

```diff
@@ -27,6 +27,7 @@ class resource_type(TypeInfo):
         name = 'StackName'
         date = 'CreationTime'
         cfn_type = config_type = 'AWS::CloudFormation::Stack'
+        universal_taggable = object()
```

Comment:
please remove this. till resource group tagging has ability to fetch for single resources, its not appropriate for large cardinality resources without a corresponding implementation for get_resources and event modes.

Reply:
oops...sorry! Addressed